# RGB image
sensor_msgs/Image rgb_image

# Depth image
sensor_msgs/Image depth_image

# Pose (position and orientation)
geometry_msgs/Pose pose